### PR TITLE
Adding the possibility to run a separate wallet

### PR DESCRIPTION
**Pull request by diofeher**
Related to #11

### CODE REVIEW

File: bitcoin/phored/wallet.go [hunk 1]
```diff
@@ -101,11 +103,16 @@ func (w *BitcoindWallet) BuildArguments(rescan bool) []string {
...
 		args = append(args, "-listen", "-proxy:127.0.0.1:"+strconv.Itoa(socksPort), "-onlynet=onion")
 	}
+	if w.separateWallet {
+		args = append(args, "-listen=0", "-datadir="+w.repoPath+"/wallet/", "-rpcuser="+connCfg.User, "-rpcpassword="+connCfg.Pass)
+	}
 	return args
 }
 
 func (w *BitcoindWallet) Start() {
-	w.shutdownIfActive()
```
Comment (meyer9) [inline bitcoin/phored/wallet.go:108]: This should shut down only if we *aren't* using a separate wallet. If we are using a separate wallet, we don't want to shut down the current one.

File: bitcoin/phored/wallet.go [hunk 2]
```diff
@@ -101,11 +103,16 @@ func (w *BitcoindWallet) BuildArguments(rescan bool) []string {
 		socksPort := defaultSocksPort(w.controlPort)
 		args = append(args, "-listen", "-proxy:127.0.0.1:"+strconv.Itoa(socksPort), "-onlynet=onion")
 	}
+	if w.separateWallet {
+		args = append(args, "-listen=0", "-datadir="+w.repoPath+"/wallet/", "-rpcuser="+connCfg.User, "-rpcpassword="+connCfg.Pass)
```
Comment (meyer9) [inline bitcoin/phored/wallet.go]: use path.Join

**Review by meyer9 — CHANGES_REQUESTED** [review]
two small changes

**Comment by diofeher** [comment]
Done! @meyer9

**Review by meyer9 — APPROVED** [review]

**Comment by meyer9** [comment]
Merged in c03e3466157ce1251053addf51c2b3447e7dcdcd. Make sure to allow edits from maintainers in the future.

**Pull request closed without merging**